examples/random_circuits_v2.py

```diff
@@ -0,0 +1,170 @@
+# Copyright 2018 The Cirq Developers
```

Comment:
"random_circuits_v2" is too ambiguous of a name. There should be enough information in the name of the method to guess at which paper this method is implementing a circuit from. e.g. "generate_google_v2_random_circuit".